cognite/client/credentials.py

```diff
@@ -35,6 +60,27 @@ def authorization_header(self) -> Tuple[str, str]:
 
 
 class OAuthClientCredentials(CredentialProvider):
+    """OAuth credential provider for the "Client Credentials" flow.
+
+    Args:
+        token_url (str): OAuth token url
+        client_id (str): Your application's client id.
+        client_secret (str): Your application's client secret
+        scopes (List[str]): A list of scopes.
```

Comment:
I have just noticed, do we miss token_custom_args here? Or is it left out on purpose? @erlendvollset